packages/runtime/src/schema.ts

```diff
@@ -178,6 +178,27 @@ const UpdateOptionsInput = new GraphQLInputObjectType({
   },
 })
 
+const CeramicSignerInputType = new GraphQLInputObjectType({
+  name: 'CeramicSignerInput',
+  fields: {
+    isAuthenticated: { type: GraphQLBoolean },
```

Comment:
What is the purpose of this input please?
Checking if the Ceramic instance has an authenticated DID should be handled internally by the context, not provided as input, if that's the goal here?